roq-core/src/main/java/org/roqmessaging/zookeeper/RoQZKHelpers.java

```diff
@@ -0,0 +1,144 @@
+package org.roqmessaging.zookeeper;
+
+import java.util.List;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.zookeeper.CreateMode;
+
+// Helper functions to wrap the zookeeper operations. 
+public class RoQZKHelpers {
+	public static boolean zNodeExists(CuratorFramework client, String path) {
+		try {
+			return client.checkExists().forPath(path) != null;
+		} catch (Exception e) {
+			return false;
+		}
+	}
+	public static void createZNode(CuratorFramework client, String path) {
+		try {
+			client.create().forPath(path);
+		} catch (Exception e) {
+			// e.printStackTrace();
+		}
+	}
+	/**
+	 * Allow to create multiple znodes 
+	 * in an atomic fashion
+	 * @param client
+	 * @param path
+	 */
+	public static void createQueueZNodes(CuratorFramework client, String queuePath,
+			String monitorPath, String monitorPL, String statMonitorPath, 
+			String statMonitorPL, String hcmPath, String hcmPL) {
+		try {
+			client.inTransaction().create().forPath(queuePath)
+				.and().create().forPath(monitorPath, monitorPL.getBytes())
+				.and().create().forPath(statMonitorPath, statMonitorPL.getBytes())
+				.and().create().forPath(hcmPath, hcmPL.getBytes())
+				.and().commit();
+		} catch (Exception e) {
+			// e.printStackTrace();
+		}
+	}
```

Comment:
I think this function should have no notion of queues, monitors or hcms. It should probably be called atomicallyCreateZNodes() and it should receive a list of nodes (path and payload) to create, using a List or using varargs (which looks like the simplest solution). What do you think?
```
public static void atomicallyCreateZNodes(CuratorFramework client, String baseNodePath, String... nodeInfo)
```